src/modules/auth/credentials/credentials.scss

```diff
@@ -9,8 +9,7 @@
 
 //CredentialCustomDataContainer
```

Comment:
@Aringoaway Btw. I believe that SCSS styling for CustomDataContainer should be in separate customdata.scss file, since credentials.scss should consist only styles for credentials components, whereas CustomDataContainer is a container, which can be used also outside of credentials.

I know that it is some residual from the past, which was forgotten here, but I still think we should do it right. I hope you dont mind to refactor also this bit within this PR :) Thank you